### PR TITLE
OSDOCS-2627: Adding admin ack to 4.9 release notes

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-2627

Preview: https://deploy-preview-36811--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-admin-ack-upgrading

**NOTE**: The xref won't work until #36809 is merged.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@wangke19 / @jianlinliu Can you please QE review this PR, which is the 4.9 release notes entry that will link to the content in PR #36809. The link at the end won't work, but that's expected. Once the previous PR merges, then the link will work.

Preview: https://deploy-preview-36811--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-admin-ack-upgrading

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -84,6 +84,19 @@ For more information, see xref:../installing/installing_aws/installing-aws-china
...
 
+[id="ocp-4-9-admin-ack-upgrading"]
+==== Required administrator acknowledgment when upgrading from {product-title} 4.8 to 4.9
+
+{product-title} 4.9 uses Kubernetes 1.22, which removed a xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-kube-1-22-apis[significant number of deprecated `v1beta1` APIs].
+
+4.8.TODO introduced a requirement that an administrator provide a manual acknowledgment before the cluster can be upgraded from {product-title} 4.8 to 4.9. This is to help prevent issues after upgrading to {product-title} 4.9, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this is done, the administrator can provide the administrator acknowledgment.
```
Comment (bobfuru) [inline release_notes/ocp-4-9-release-notes.adoc:92]: Similar comment to the related PR, here I'd suggest saying, "...that an administrator must provide..."

**Review by bobfuru — COMMENTED** [review]
One minor nit but otherwise LGTM!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (jianlinliu) [reply release_notes/ocp-4-9-release-notes.adoc:92]: Agree, and sounds like the last sentence is duplicated with the following 96 line.

**Comment by jianlinliu** [comment]
Only a minor comments, others looks good to me.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-9-release-notes.adoc:92]: Thanks! Will update to add that, and I'll fix the last sentence to be more in line w/ how i ended it with the kbase article

**Comment by bergerhoffer** [comment]
Updated PR to use 4.8.14 as the expected z-stream. Will submit a separate PR to update if it ends up being a different version.

**Comment by bergerhoffer** [comment]
Updated per suggestions, merging. As mentioned, if the z-stream ends up being a different number, will submit another PR to update that

**Pull request merged**